dts/arm/silabs/efr32bg22.dtsi

```diff
@@ -54,11 +54,18 @@
 		};
 
 		usart0: usart@5005c000 {
-			compatible = "silabs,gecko-usart";
+			compatible = "silabs,gecko-spi-usart";
 			reg = <0x5005C000 0x400>;
 			interrupts = <13 0>, <14 0>;
 			interrupt-names = "rx", "tx";
 			status = "disabled";
+
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			cs-gpios = <&gpioc 3 GPIO_ACTIVE_LOW>;
```

Comment:
is this GPIO connected internally at SoC level?

Reply:
this refers to SoC level GPIO controller

Reply:
this refers to a specific GPIO pin, not only the controller. Is it hard-wired in the SoC?

Reply:
CS is connected at the board level - moved `cs-gpios` to board-specific DTS.